### PR TITLE
Update introduction.md

### PR DESCRIPTION
**Pull request by mswezey23**
`use version version 1.0.0-beta.36 or revert`

### CODE REVIEW

File: docs/ethereum/wss/introduction.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Some tools you can use for making these requests
 The following is an example showing a connection to the websocket endpoint and using subscriptions through web3.js 1.0.
 
 NOTE: web3.js 1.0.0-beta.34 has an open issue with request headers. (https://github.com/ethereum/web3.js/issues/1559)
-Users will have to revert to version 1.0.0-beta.33 to avoid this issue.
+Users will have use version version 1.0.0-beta.36 or revert to version 1.0.0-beta.33 to avoid this issue.
```
Comment (ryanschneider) [inline docs/ethereum/wss/introduction.md]:
```suggestion
Use version 1.0.0-beta.36 or newer or revert to version 1.0.0-beta.33 to avoid this issue. 
```

Comment (ryanschneider) [reply docs/ethereum/wss/introduction.md]: Typo in "use version version", offered a suggestion on a more concise sentence overall.

**Comment by captDaylight** [comment]
@mswezey23 once you make that update I'll merge it in.

**Comment by ncocchiaro** [comment]
We'll add this on our end, thanks!

**Pull request closed without merging**